app/assets/javascripts/components/story/ExpandedStory/ExpandedStoryState.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import * as Story from '../../../models/beta/story';
+
+const ExpandedStoryState = (props) => {
```

Comment:
`({ story, onEdit })` looks fancier 👌

Reply:
fancier is the best expression ever! :smile: